ToGather/ToGather/Screens/SavingRecord/View/SavingRecordView.swift

```diff
@@ -97,22 +114,27 @@ struct SavingRecordView: View {
             }
             .padding(.bottom, 16)
             Button(action: {
+                guard let _ = image else {
+                    return
+                }
+                // image 파일이 존재할 때 Firebase에 쓰는 기능
                 print("이번주 저축 완료하기")
             }, label: {
                 Text("이번주 저축 완료하기")
                     .fontWeight(.bold)
                     .frame(width: UIScreen.main.bounds.width - 40, height: 46)
```

Comment:
이 방법도 좋구
```suggestion
            .frame(minWidth: 0, maxWidth: .infinity, minHeight: 46)
            .padding(horizontal)
```
이러한 방법도 소개해봅니당!

Reply:
좋은 표현인 것 같아요! 저는 UIScreen이라는 상단에 있는 화명크기로 잡았다면 추천해주신 방법은 부모 뷰와의 상대적인 값으로 Frame을 정해서 더 안전하고 직관적인것 같아요! 수정하겠습니다.